minio/minio
# Add support for Object Tagging in LifeCycle configuration

## Description
This PR adds support for Object Tagging in LifeCycle configuration

## Motivation and Context
See #8870 

## How to test this PR?
- Add LifeCycle configuration with Object Tag fields.
- Upload objects with Object Tags
- Check if relevant objects get expired as configured in LifeCycle config

## Types of changes

## Checklist:
